src/sentry/models/grouphash.py

```diff
@@ -23,3 +24,38 @@ class Meta:
         app_label = 'sentry'
         db_table = 'sentry_grouphash'
         unique_together = (('project', 'hash'),)
+
+    @staticmethod
+    def fetch_last_processed_event_id(project_id, group_hash_ids):
+        prefix = 'last-processed-event:{}'.format(project_id)
+        with redis.clusters.get('default').map() as client:
```

Comment:
This should probably be targeted to a single host based on the project ID so this isn't as exposed to partial failures.